### DISCUSSION TITLE
Minder Release Announcements and Calls for Translation

### DISCUSSION THREAD

**Discussion by phase1geo** [phase1geo/Minder#409] [Show and tell]
# What is this discussion board about?

This discussion will act as an announcement board for upcoming Minder releases and calls for translators to update any necessary translations to go into that release.

**If you are a Minder translator or have interest in translating Minder into your language, please follow/subscribe to this discussion thread.**

**Comment by phase1geo** [comment]
## Call for Minder 1.14.0 translations!

I am planning on releasing the 1.14.0 release on Sunday (1/16/2022).  Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.15.0 translations!

I am planning on releasing the 1.15.0 release on Sunday (4/23/2023). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.15.2 translations!

I am planning on releasing the 1.15.2 release on Monday (5/22/2023). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.15.3 translations!

I am planning on releasing the 1.15.3 release on Tuesday (8/22/2023). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.16.0 translations!

I am planning on releasing the 1.16.0 release on Sunday (1/7/2024). Please have your translation pull requests submitted by 8pm (CST).

**Comment by prokoudine** [comment]
> I am planning on releasing the 1.16.0 release on Sunday (1/7/2024). 

How do I access summary nodes?

**Reply by phase1geo** [reply]
Summary node support is not going to be included in 1.16.0.  There are issues with layout that may take some time to fix.  I don't want to hold up the rest of the changes/fixes for the completion of this feature.

**Comment by phase1geo** [comment]
## Call for Minder 1.16.2 translations!

I am planning on releasing the 1.16.2 release on Monday (10/15/2024). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.16.3 translations!

I am planning on releasing the 1.16.3 release on Sunday (2/11/2024). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.16.4 translations!

I am planning on releasing the 1.16.4 release on Monday (4/8/2024).  Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Minder 1.17.0 translations!

I am planning on releasing the 1.17.0 release on Wednesday (10/30/2024). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Minder 2.0.0-beta1 Released

After a long development period, we are proud to announce the first beta version of Minder 2.0.  Due to the extent of the changes between Minder 1.17 and 2.0, we are trying out a beta path of releases to allow people to try things out and file issues before officially releasing 2.0.

This is also a call to update translations for Minder 2.0.  There is a lot of textual changes within the application that need translating.  Feel free to create pull requests for these changes and I will get them in.

New beta releases will happen often between now and the official 2.0 version.  I am primarily interested in fixing bugs and improving translations during the beta period.  New feature requests will be put off until after 2.0 has been released.

## Important Note

An important note about Minder 2.0, the .minder file format has changed such that files saved under Minder 2.0 will not be editable/readable by Minder 1.x.  Minder 2.0 will detect a 1.x formatted file and give you the option of what to do with it.  There are three options:

1. Upgrade the file to 2.0.
2. Save the 1.x file with a modified name and upgrade the file to 2.0.
3. Load the 1.x file in a readonly state (it cannot be modified).

During the beta period, if you are attempting to modify an important mind map to you, I would suggest using option 2 above.  That way if something goes wrong, you will still have the 1.x file around to using in your non-beta version of Minder.



Have fun!
